app/assets/locales/messages_ccodk_default.txt

```diff
@@ -19,7 +19,7 @@ verify.progress = Verifying resources. ${0} resources verified of ${1} total
 
 modules.m0=pregenancy
 
-install.barcode=Welcome to CommCare! Please choose an installation method below 

```

Comment:
hmm this was originally changed before the merge, so reverting would also mean reverting a lot of changes since (including the localization merges). Reverting that hunk isn't working either. Not sure why this is showing up as a diff at all